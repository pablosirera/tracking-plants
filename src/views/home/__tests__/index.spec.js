import { shallowMount } from '@vue/test-utils'
import HomeView from '../index.vue'

describe('HomeView', () => {
  const build = () => {
    const wrapper = shallowMount(HomeView)

    return {
      wrapper
    }
  }
  it('should match snapshot', () => {
    const { wrapper } = build()
    expect(wrapper.element).toMatchSnapshot()
  })
})
