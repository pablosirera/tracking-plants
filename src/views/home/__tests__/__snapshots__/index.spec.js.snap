// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeView should match snapshot 1`] = `
<div
  class="home"
>
  <listplants-stub />
</div>
`;
